resources/buildPlatformMap.properties

```diff
@@ -35,6 +35,9 @@ linux_ppc=ppc32_linux
 linux_riscv64_nocmprssptrs=riscv64_linux_xl
 linux_riscv64_cmprssptrs=riscv64_linux_cmprssptrs
 linux_riscv64=riscv64_linux
+linux_loongarch64_nocmprssptrs=loongarch64_linux_xl
+linux_loongarch64_cmprssptrs=loongarch4_linux_cmprssptrs
```

Comment:
looks like `loongarch4_linux_cmprssptrs` should be `loongarch64_linux_cmprssptrs`